lyonza/WhiteTextBroadcast.py

```diff
@@ -256,11 +256,11 @@ def _add_series_count_text_no_season(self, titled_image: Path) -> Path:
             f'convert "{titled_image.resolve()}"',
             *self.__series_count_text_global_effects(),
             f'-font "{self.EPISODE_COUNT_FONT.resolve()}"',
-            f'-gravity center',
+            f'-gravity northwest',
             *self.__series_count_text_black_stroke(),
-            f'-annotate -1400-750 "{self.episode_text}"',
+            f'-annotate "{self.episode_text}"',
             *self.__series_count_text_effects(),
-            f'-annotate -1400-750 "{self.episode_text}"',
+            f'-annotate "{self.episode_text}"',

```

Comment:
```suggestion
            f'-annotate +25+25 "{self.episode_text}"',
```
You need to specify some sort of offset (probably not +25+25) for the text

Reply:
I updated my WhiteTextAbsolute.pay yesterday you can find the new offset there.